setup.py

```diff
@@ -44,6 +44,6 @@
       author_email='info@quantumsimulations.de',
       url='https://quantumsimulations.de',
       license=License,
-      python_requires='>=3.8.*, !=3.9.*, !=3.11.*',
+      python_requires='>=3.8.*,  !=3.11.*',  # !=3.9.*,
```

Comment:
It used to be that python 3.9 was not supported for my_qlm, I'm not sure why. But if that restriction has been lifted, wonderful!